sdk/compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -621,6 +622,19 @@ buildLfPackageGraph =
               , Just quals <- [LFC.decodeModuleImports ty]
               , LF.Qualified { LF.qualPackage } <- Set.toList quals
               ]
+              -- Pull out all package ids from re-exports as well
```

Comment:
Should we also ignore names that are not explicitly exported as an optimization?

Reply:
We can't, since even if they're not exported, they are still needed to compile "the current package"
This logic is for listing the dependents of a package, regardless of what this package exports. We simply didn't previously consider re-exports as a dependent, which is what this fixes.